cmake/Install.cmake

```diff
@@ -266,8 +298,30 @@ if( APPLE )
   install(PROGRAMS scripts/runepmacro DESTINATION "./")
   install(PROGRAMS scripts/runreadvars DESTINATION "./")
 
+  # You need at least one "install(..." command for it to be registered as a component
   install(CODE "MESSAGE(\"Creating symlinks.\")" COMPONENT Symlinks)
```

Comment:
Could we change this from something besides a MESSAGE?  I'm not sure what is closer to a no-op, but this is showing up in the CI output parsing.  If you look at the package warnings [here](http://nrel.github.io/EnergyPlusBuildResults/EnergyPlus-Test20190717b-x86_64-MacOS-10.13-clang.html#package) you'll see Creating Symlinks showing up.  It's not causing a problem in the output generally speaking, but when CI is trying to parse errors on the output, it's getting in the mix.

Reply:
Hum, that was my no-op option, and it also displays during actual install which is nice. Cf https://cmake.org/cmake/help/v3.13/command/install.html#custom-installation-logic

Perhaps message(TRACE .. would work since it sends to stdout instead of stderr. If you can make that change and push to this PR it'd be helpful because i won't have to wait tomorrow @Myoldmopar